### DISCUSSION TITLE
Change date format

### DISCUSSION THREAD

**Discussion by nektariosdi** [Leantime/leantime#488] [Q&A]
Hello, how could I change the date/time format?

**Comment by marcelfolaron** [comment]
Right now the date format can only be changed in the language files in resources/language/ (lines 4-8)
You can also copy the language file and give it a different name "en-US1.ini" for example to make sure your changes will be retained after an update.

**Comment by nektariosdi** [comment]
Thank you so much!
One more question, Edit user option doesn't seem to work. Am I wrong?

**Comment by marcelfolaron** [comment]
Yes, there is a bug in 2.8. A fix has been submitted and will be included
in the next release. I am fixing some other parts but gope to get this out
by the end of the week.

**Comment by michael-schramm** [comment]
Hi,
I am using German as the default language but still I get "AM/PM" time format shown, i.e. when creating "Events".
How can I fix this?

**Comment by marcelfolaron** [comment]
Let me check if that is a bug. That should not be happening

**Reply by michael-schramm** [reply]
![image](https://user-images.githubusercontent.com/45664277/179758793-8927ceaf-88df-4636-9ac8-bcbfc3b5aa4b.png)

Here is a screenshot.
